.github/workflows/build.yaml

```diff
@@ -43,8 +43,8 @@ jobs:
           major=${branch%%.*}
           minor=${branch#*.}
           attr=php$major$minor
-          echo "::set-output name=major::$major"
-          echo "::set-output name=attr::$attr"
+          echo "major=php$major$minor" >> $GITHUB_OUTPUT
```

Comment:
This should be just `$major`

Reply:
Also would be nice to link https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/ in the commit message.

Reply:
Oh nice catch!

I fixed it from my phone, feel free to rebase or I'll do it tomorrow

Reply:
Fixed!